main.js

```diff
@@ -0,0 +1,339 @@
+if (a.length < 5) {
+  console.log(a,"has less than 5 characters");
+} else {
+  console.log(a,"has 5 characters or more");
+}
+
+console.log(`Variables 2.2:`)
+if (b > 0 && b < 5) {
+  console.log("ping");
+} else { 
+  console.log("pong");
+}
+
+console.log(`Variables 2.3:`)
+if (c > b) {
+  console.log(`${c} is larger than ${b}`);
+} else { 
+  console.log(`${c} is smaller than ${b}`);
+}
+
+// functions
+
+// 1. Write a function that will take 2 numbers as inputs, then return the sum of the 2 numbers.
+//function add() {}
+function add(a,b) {
+  return a + b;
+}
+document.getElementById("f1").innerHTML = `Function 1 (${b} + ${c}) = ` + add(b,c);
+
+// 2. Create a similar function as the above, but return the multiple of the 2 numbers.
+//function multiply() {}
+function multiply(a,b) {
+  return a * b;
+}
+document.getElementById("f2").innerHTML = `Function 2 (${b} * ${c}) = ` + multiply(b,c);
+
+// 3. Write a function that accepts a number as input, if the number is odd, return a string 'odd', if this number is even, return a string 'even'.
+//function oddOrEven() {}
+
+function oddOrEven(x) {
+    if (x % 2 != 0) {
+    return `Odd`;
+  } else {
+    return `Even`;
+  } 
+}
+document.getElementById("f3").innerHTML = `Function 3: is ${b} odd or even? ` + oddOrEven(b)
+
+// 5. Fix this function. We want to see 2 in the console instead of undefined
+//function hoisting() {
+//  console.log(y) // undefined
+// let y = 2
+//}
+
+console.log(`Function 5:`);
+function hoisting() {
+  let y = 2
+  console.log(y) // undefined
+}
+document.getElementById("f5").innerHTML = `Function 5: check console.`;
+hoisting()
+
+
+// 6. Write a count function that runs from 1 to 100 using for loop
+// if the current count is odd, print "odd", else print "even"
+
+let count = "";
+for (let i = 1; i < 101; i++) {
+  count =  count + i + ` ` + oddOrEven(i) + "<br>";
+}
+document.getElementById("f6").innerHTML = `Function 6:` + "<br>" + count
+
+// 7. Write a function that accepts a string as an input, and return a new string with the first character uppercased
+// Ex: capitalized("abc") will return "Abc"
+//function capitalized() {}
+
+function myFunction() {
+  let text = document.getElementById("demo").innerHTML;
+  document.getElementById("demo").innerHTML =
+  text.toUpperCase();
+}
+
+let text = "abc";
+function capitalized(x){
+ return x[0].toUpperCase() + x.substring(1);
+}
+document.getElementById("f7").innerHTML = `Function 7: ` + capitalized(text);
+
+// 8. Write a function accepts a number as input and then return the number reversed
+// Ex: reverse(1234) will return 4321
+//function reverse() {}
+
+let number = 1234;
+function reverse(x) {
+  let toS = x.toString();
+  //console.log(toS);
+  let s = toS.split();
+  //console.log(s);
+  const splits = Array.from(toS);
+  let inv = splits.reverse();
+  //console.log(`inversed : ` + inv.join());
+  let join = inv.join();
+  let replace = join.replace(/,/g,"");
+  return replace
```

Comment:
Is it possible to code it in a simpler way?

Reply:
`function reverse(i) {
	i = i + ""
  return i.split("").reverse().join("")
}
console.log(Number(reverse(53827)))`

I think this is shorter?